app/src/main/java/com/garethevans/church/opensongtablet/songmenu/SongMenuFragment.java

```diff
@@ -656,10 +647,36 @@ public void moveToSongInMenu(Song song) {
                 if (songListLayoutManager!=null) {
                     handler.post(() -> {
                         try {
-                            songListLayoutManager.scrollToPositionWithOffset(songListAdapter.getPositionOfSong(song),0);
+                            // IV - On song change, like v5, use a song list filtered by folder only
```

Comment:
Why do we clear the users preferred song list preferences?  v5 didn't have these search filter options, so not sure why we now revert to this?
Some users use a TAG to show only some songs to choose from

Reply:
See note.

Reply:
Removed filter reset.  Please consider https://github.com/thebigg73/OpenSongTablet/pull/238/commits/7fbd41dbe6e75898504f0c6f11d3542455da925f
commit.